nerdtree_plugin/fs_menu.vim

```diff
@@ -203,7 +203,7 @@ endfunction
 function! NERDTreeListNode()
     let treenode = g:NERDTreeFileNode.GetSelected()
     if treenode != {}
-        let metadata = split(system('ls -ld ' . shellescape(treenode.path.str())), '\n')
+        let metadata = split(system('ls -ldh ' . shellescape(treenode.path.str())), '\n')
```

Comment:
would be possible to use stat and ls -lh ? 
```
stat -f "%z" "2017-02-14-Windsor-T21986.jpeg"
823989
``` 
returns only the file size in bytes. but not sure if the formatting options are consistent accros OSX and Linux